taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -99,37 +121,51 @@ const StartProject = ({ clearNavBarLinks }) => {
           </FormControl>
         </div>
 
-        <div>
-          <TextField
-            error={isInvalidBranchName}
-            helperText={
-              isInvalidBranchName &&
-              "Special characters, capital letters and white spaces are not allowed"
-            }
-            size="small"
-            sx={{ width: 265, mt: 2 }}
-            onChange={(event) => {
-              setBranchName(event.target.value);
-            }}
-            value={branchName}
-            variant="outlined"
-            label="Branch Name"
-          />
-        </div>
+        <TextField
```

Comment:
We shall add a description to explain:
Please use your Github account username if possible, or eventually your id on open food facts slack (so that we can contact you)